magenta/interfaces/midi/magenta_midi.py

```diff
@@ -89,7 +89,9 @@
 tf.app.flags.DEFINE_integer(
     'temperature_control_number',
     None,
-    'The control change number to use for controlling softmax temperature.')
+    'The control change number to use for controlling softmax temperature.'
+    'This should be an integer between 0 and 127 corresponding to a softmax '
+    'temperature between 0.1 and 2.')
```

Comment:
Re-reading this, I realize that it is actual not correct. It should really say something like 'The value of control changes using this control number will be used to set the temperature in a range between 0.1 and 2.'